### PR TITLE
[READY FOR REVIEW] PR for video events support issue

### LINKED ISSUE #1446: Add support for video events
We currently only have event types for file and image, but we should also have a video type to handle things like mp4.

### PR DESCRIPTION
**Pull request by arctdav**
# Description

Fixes #1446

## Status
**READY** | UNDER DEVELOPMENT | ON HOLD

## Type of change
- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

# How Has This Been Tested?

- Not yet tested

# Checklist:

- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation (if applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by arctdav** [comment]
What feature should the Video event type support? I suppose it's not the same feature as image dimension.

**Comment by arctdav** [comment]
any updates?

**Comment by arctdav** [comment]
@Cadair , I created a class for Video representation. Used BitArray to convert bytes into binary/hex. Let me know if you need more feature for Video. I feel there lacks support for interpreting a video directly from binary in Python. May need to implement a binary interpreter for video, if binary video data support not found in Python3.

File: opsdroid/events.py [hunk 1]
```diff
@@ -402,6 +403,22 @@ async def get_dimensions(self):
         fbytes = await self.get_file_bytes()
         return get_image_size_from_bytesio(io.BytesIO(fbytes), len(fbytes))
 
+class Video(File):
+    """Event class specifically for video files."""
+    async def get_bitarray(self):
+        """
```
Comment (FabioRosado) [inline opsdroid/events.py:409]: This docstring will fail, the method needs to look something like this:
```suggestion
        """Return a bitarray of the video bytes.
        This method enable video bytes to be converted to hex/bin.
        Doc: https://github.com/scott-griffiths/bitstring/blob/master/doc/bitarray.rst
        """
```

File: requirements.txt [hunk 2]
```diff
@@ -26,4 +26,8 @@ ibm-watson==4.4.0
 websockets==8.1
 webexteamssdk==1.3
 voluptuous==0.11.7
+<<<<<<< HEAD
```
Comment (FabioRosado) [inline requirements.txt:29]: It seems you had git conflicts when you pushed your commit. Not sure if you know how to deal with git conflicts but I'm happy to give you a hand with it if you need, just let me know.

In this case you can quickly fix the issue by removing the `<<<<<<< HEAD`, `=======`, `>>>>>>> 48e61c4ed1b1ccbb146a765170a5ce1d50fa1384`
```suggestion
bitstring==3.1.7 
regex==2020.4.4
```

**Comment by arctdav** [comment]
@FabioRosado could you direct me to some good test example previous pull requests has done?

File: requirements.txt [hunk 3]
```diff
@@ -26,4 +26,9 @@ ibm-watson==4.4.0
 websockets==8.1
 webexteamssdk==1.3
 voluptuous==0.11.7
+bitstring==3.1.7 
 regex==2020.4.4
+bitstring==3.1.7 
+=======
```
Comment (FabioRosado) [inline requirements.txt:32]: I believe I didn't explain well what you need to do here, but you need to remove this line 
```suggestion

```

File: requirements.txt [hunk 4]
```diff
@@ -26,4 +26,9 @@ ibm-watson==4.4.0
...
+regex==2020.4.4
+>>>>>>> 48e61c4ed1b1ccbb146a765170a5ce1d50fa1384
```
Comment (FabioRosado) [inline requirements.txt:34]: You need to remove this line as well 😄 
```suggestion

```

**Comment by FabioRosado** [comment]
Hello @arctdav  how are you doing? Are you able to pick up on this PR and write the tests for it? Do you need help?

**Comment by arctdav** [comment]
@Rosado, I’ll do it once I’m available. Busy with school work now.

**Pull request closed without merging**